src/features/common/types/user.d.ts

```diff
@@ -17,3 +18,33 @@ export interface PublicUser {
   /** @deprecated do not use */
   hasLogoLicense?: boolean;
 }
+
+export interface ContributionType {
+  type: string;
+  geometry: Geometry;
+  properties: Properties;
+}
+interface Properties {
+  id: string;
+  type: string;
+  plantDate?: string;
+  country: string;
+  images: Image[];
+  treeCount: number;
+  unitType: string;
+  project: Project;
+  recipient?: Owner | null;
+  giver?: Owner | null;
```

Comment:
The type may be identical, but `Owner` does not seem to be an appropriate name for the type used both by recipient and giver. 

Perhaps `Owner` could be renamed to `PersonOrOrganization` or `AvatarType` or something similar and updated in all places?